### PR TITLE
fix(ui) - ENTESB-14905 - SOAP address not showing in detail page

### PR DESCRIPTION
**Pull request by kahboom**
fix for [ENTESB-14905](https://issues.redhat.com/browse/ENTESB-14905):

- hide all fields on Detail page Edit view that don't contain an existing value
- show the Address field if a value for it exists

Screen Shot 2020-10-06 at 15 15 12

Screen Shot 2020-10-06 at 15 15 19

### CODE REVIEW

File: app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailConfigEdit.tsx [hunk 1]
```diff
@@ -56,68 +58,92 @@ export const ApiConnectorDetailConfigEdit: React.FunctionComponent<IApiConnector
...
+            id="connector-name"
+            aria-describedby="horizontal-form-name-helper"
+            data-testid={'api-connector-name-field'}
+            name="name"
+            onChange={onChange}
+            validated={isValid}
+          />
+        </FormGroup>
+      )}
+      {properties.description && (
```
Comment (kahboom) [inline app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailConfigEdit.tsx:82]: What a pain that the formatting has caused the diff to look like this.. all I did was add a check for the property like `{properties.description && ( // display the field )}`, and of course add a form group for the Address property.

Comment (gashcrumb) [reply app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailConfigEdit.tsx:82]: np :smile:

**Review by gashcrumb — APPROVED** [review]
:+1:

**Comment by gashcrumb** [comment]
@kahboom looks like some of the test code needs to be updated with that interface change.

**Review by gashcrumb — APPROVED** [review]
:+1

**Pull request merged**